docs/database/use-entity-framework-db-contexts.md

```diff
@@ -0,0 +1,564 @@
+
+    ```csharp
+    builder.Services.AddDbContextPool<ExampleDbContext>(options =>
+        options.UseSqlServer(builder.Configuration.GetConnectionString("database")
+            ?? throw new InvalidOperationException("Connection string 'database' not found.")));
+    ```
+
+    :::zone-end
+    :::zone pivot="postgresql-ef"
+
+    ```csharp
+    builder.Services.AddDbContextPool<ExampleDbContext>(options =>
+        options.UseNpgsql(builder.Configuration.GetConnectionString("database")
+            ?? throw new InvalidOperationException("Connection string 'database' not found.")));
+    ```
+
+    :::zone-end
+    :::zone pivot="oracle-ef"
+
+    ```csharp
+    builder.Services.AddDbContextPool<ExampleDbContext>(options =>
+        options.UseOracle(builder.Configuration.GetConnectionString("database")
+            ?? throw new InvalidOperationException("Connection string 'database' not found.")));
+    ```
+
+    :::zone-end
+    :::zone pivot="mysql-ef"
+
+    ```csharp
+    builder.Services.AddDbContextPool<ExampleDbContext>(options =>
+        options.UseMySql(builder.Configuration.GetConnectionString("database")
+            ?? throw new InvalidOperationException("Connection string 'database' not found.")));
+    ```
+
+    :::zone-end
+
+- Call the EF <xref:Microsoft.Extensions.DependencyInjection.EntityFrameworkServiceCollectionExtensions.AddPooledDbContextFactory*> method instead of the EF <xref:Microsoft.Extensions.DependencyInjection.EntityFrameworkServiceCollectionExtensions.AddDbContextFactory*> method.
+
+    :::zone pivot="sql-server-ef"
+
+    ```csharp
+    builder.Services.AddPooledDbContextFactory<ExampleDbContext>(options =>
+        options.UseSqlServer(builder.Configuration.GetConnectionString("database")
+            ?? throw new InvalidOperationException("Connection string 'database' not found.")));
+    ```
+
+    :::zone-end
+    :::zone pivot="postgresql-ef"
+
+    ```csharp
+    builder.Services.AddPooledDbContextFactory<ExampleDbContext>(options =>
+        options.UseNpgsql(builder.Configuration.GetConnectionString("database")
+            ?? throw new InvalidOperationException("Connection string 'database' not found.")));
+    ```
+
+    :::zone-end
+    :::zone pivot="oracle-ef"
+
+    ```csharp
+    builder.Services.AddPooledDbContextFactory<ExampleDbContext>(options =>
+        options.UseOracle(builder.Configuration.GetConnectionString("database")
+            ?? throw new InvalidOperationException("Connection string 'database' not found.")));
+    ```
+
+    :::zone-end
+    :::zone pivot="mysql-ef"
+
+    ```csharp
+    builder.Services.AddPooledDbContextFactory<ExampleDbContext>(options =>
+        options.UseMySql(builder.Configuration.GetConnectionString("database")
+            ?? throw new InvalidOperationException("Connection string 'database' not found.")));
+    ```
+
+    :::zone-end
+
+Remember to enrich the database context after using the last two methods, as described above.
+
```

Comment:
Note that only the base context state is reset when it's returned to the pool. If you've manually changed the state of the `DbConnection` or another service, you must also manually reset it. Additionally, context pooling prevents you from using `OnConfiguring` to configure the context. See [DbContext pooling](/ef/core/performance/advanced-performance-topics#dbcontext-pooling) for more information.

Reply:
Fixed.